install/kubernetes/helm/istio/values-istio-grafana-config.yaml

```diff
@@ -0,0 +1,544 @@
+# Top level istio values file has the following sections.
+#
+# global: This file is the authoritative and exhaustive source for the global section.
+#
+# chart sections: Every subdirectory inside the charts/ directory has a top level
+#       configuration key in this file. This file overrides the values specified
+#       by the charts/${chartname}/values.yaml.
+#       Check the chart level values file for exhaustive list of configuration options.
+
+#
+# Gateways Configuration, refer to the charts/gateways/values.yaml
+# for detailed configuration
+#
+gateways:
+  enabled: true
+
+#
+# sidecar-injector webhook configuration, refer to the
+# charts/sidecarInjectorWebhook/values.yaml for detailed configuration
+#
+sidecarInjectorWebhook:
+  enabled: true
+
+#
+# galley configuration, refer to charts/galley/values.yaml
+# for detailed configuration
+#
+galley:
+  enabled: true
+
+#
+# mixer configuration
+#
+# @see charts/mixer/values.yaml, it takes precedence
+mixer:
+  policy:
+    # if policy is enabled the global.disablePolicyChecks has affect.
+    enabled: true
+
+  telemetry:
+    enabled: true
+#
+# pilot configuration
+#
+# @see charts/pilot/values.yaml
+pilot:
+  enabled: true
+
+#
+# security configuration
+#
+security:
+  enabled: true
+
+#
+# nodeagent configuration
+#
+nodeagent:
+  enabled: false
+
+#
+# addon grafana configuration
+#
+grafana:
+  enabled: true
+  
+  persist: true # 
+  
+  security: #
+    secretName: grafana-secrets #
+    usernameKey: ADMIN_USERNAME #
+    passphraseKey: ADMIN_PASSWORD #
+
+  env: # {}
+    # env_variable_name: value
+    GF_SMTP_ENABLED: true
```

Comment:
I feel like this should default to false and the options below should just be examples.

Reply:
Indeed, I commited it with demo values. Will comment out and add notes accordingly. Thanks

Reply:
Moved the default values and also commented out example values to the install/kubernetes/helm/istio/charts/grafana/values.yaml file